src/client/NuGet.Client.Staging/StageViews.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace NuGet.Client.Staging
+{
+    public class StageDetailedView : StageView

```

Comment:
moved to one class per file. A hierarchy exists between the classes. What do u mean?

---
In reply to: [66830236](https://github.com/NuGet/NuGet.Staging/pull/62#discussion_r66830236) [](ancestors = 66830236)